libs/labelbox/tests/data/annotation_import/test_bulk_import_request.py

```diff
@@ -219,110 +219,3 @@ def test_delete(module_project, predictions):
     bulk_import_request.delete()
     all_import_requests = module_project.bulk_import_requests()
     assert len(list(all_import_requests)) == 0
-
-
-def _create_label(row_index, data_row_uids, label_name_ids=['bbox']):
-    label_name = label_name_ids[row_index % len(label_name_ids)]
-    data_row_uid = data_row_uids[row_index % len(data_row_uids)]
-    return Label(data=GenericDataRowData(uid=data_row_uid),
-                 annotations=[
-                     VideoObjectAnnotation(name=label_name,
-                                           keyframe=True,
-                                           frame=4,
-                                           segment_index=0,
-                                           value=Rectangle(
-                                               start=Point(x=100, y=100),
-                                               end=Point(x=105, y=105),
-                                           ))
-                 ])
-
-
-@pytest.mark.parametrize("configured_project", [MediaType.Video], indirect = True)
-@patch('labelbox.schema.annotation_import.ANNOTATION_PER_LABEL_LIMIT', 20)
-def test_below_annotation_limit_on_single_data_row(
```

Comment:
Remove this we are constantly testing for this case everytime we do a label import